core-services/ci-chat-bot/workflows-config.yaml

```diff
@@ -117,7 +117,7 @@ workflows:
     platform: aws
   cucushift-installer-rehearse-aws-ipi-proxy-techpreview:
     platform: aws
-  cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8:
+  cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips:
```

Comment:
I think we do not need to change it any more, if want a quick merge, I think we can revert the change on this file.